test/e2e/common/setup-kubestellar.sh

```diff
@@ -98,19 +119,40 @@ case "$CLUSTER_SOURCE" in
         ;;
 esac
 
-
 :
 : -------------------------------------------------------------------------
-: Install the core-chart
+: Install the core-chart with scenario-specific configuration
 :
 
 pushd "${SRC_DIR}/../../.."
+
+# Prepare ITS and WDS configurations based on scenario
+ITS_CONFIG=""
+WDS_CONFIG=""
+
+if [ "$config_combined_control_plane" = "true" ]; then
+    ITS_CONFIG='[{"name":"combined-cp"}]'
+    WDS_CONFIG='[{"name":"combined-cp"}]'
+else
+    if [ "$config_wds_type" = "hosting" ]; then
+        WDS_CONFIG='[{"name":"wds1","hosting":true}]'
+    else
+        WDS_CONFIG='[{"name":"wds1"}]'
+    fi
+
+    if [ "$config_its_type" = "hosting" ]; then
+        ITS_CONFIG='[{"name":"its1","hosting":true}]'
```

Comment:
If I recall correctly, `"hosting":true` is not the way to get this done. See https://github.com/kubestellar/kubestellar/blob/8b33007d864576b18379751433c4d8ce720ffa7d/core-chart/values.yaml#L108 ,